bindings/ir/operations/register_optype.cpp

```
@@ -10,65 +10,226 @@
-      .value("rzz", qc::OpType::RZZ)
-      .value("rzx", qc::OpType::RZX)
-      .value("xx_minus_yy", qc::OpType::XXminusYY)
-      .value("xx_plus_yy", qc::OpType::XXplusYY)
-      .value("compound", qc::OpType::Compound)
-      .value("measure", qc::OpType::Measure)
-      .value("reset", qc::OpType::Reset)
-      .value("barrier", qc::OpType::Barrier)
-      .value("if_else", qc::OpType::IfElse)
-      .export_values()
-      .finalize();
+void registerOptype(const nb::module_& m) {
```

Cppcheck (2.18.0)

[style] The function 'registerOptype' is never used.

(unusedFunction)